src/modules/shmem/consumer_shmem.yml

```diff
@@ -0,0 +1,34 @@
+schema_version: 0.2
+type: consumer
+identifier: shmem
+title: Shared memory output
+version: 3
+copyright: Copyright (C) 2016 Meltytech, LLC
```

Comment:
The code is copyright you, which justifies to put it in a separate module. Thus, you should change copyright to yourself in the yml. Also, consider to set the version to 1 (unless you need it this level for your environment). Same applies to the producer yml.